tests/utils.py

```diff
@@ -292,7 +293,7 @@ def standard_create(stream):
         )
     elif stream == 'payouts':
         return client[stream].create(
-            amount=random.randint(0, 10000),
+            amount=random.randint(1, 10),
```

Comment:
Sometimes `randint(0, 10000)` return 0 value, also. But, the `amount` field value must be greater than or equal to 1. ([Circleci Failure](https://app.circleci.com/pipelines/github/singer-io/tap-stripe/1149/workflows/f5ae7561-e652-4f09-86ed-257d47cfb3bb/jobs/2755))